nightwatch.conf.js

```diff
@@ -0,0 +1,152 @@
+'use strict'
+
+const fs = require('fs');
```

Comment:
Is this part of the demo? It looks like a very elaborated nightwatch configuration. If it is part of the demo, please put it in the demo directory. It'd be convenient to simplify it in order to make it easier to digest for the ones learning how to use this utility.